user_guides/jakartaee/src/main/jbake/content/jaspic-files.adoc

```diff
@@ -2,7 +2,7 @@ type=page
 status=published
 title=Jakarta Authentication Technology Notes and Files
 next=database-config.html
-prev=csiv2logs.html
+prev=commonappdeploy.html
 ~~~~~~
```

Comment:
^ is updating the doc that was after CSIv2 doc to reference the correct previous doc (commonappdeploy) the same as mentioned above.